Framework/CurveFitting/src/Functions/Activation.cpp

```diff
@@ -0,0 +1,84 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidCurveFitting/Functions/Activation.h"
+#include "MantidAPI/FunctionFactory.h"
+#include "MantidKernel/PhysicalConstants.h"
+#include <boost/algorithm/string.hpp>
+
+#include <cmath>
+
+namespace Mantid::CurveFitting::Functions {
+
+using namespace CurveFitting;
+
+using namespace Kernel;
+
+using namespace API;
+
+DECLARE_FUNCTION(Activation)
+
+void Activation::init() {
+  declareAttribute("Unit", Attribute("K")); // or meV
+  declareParameter("AttemptRate", 1000.0, "coefficient for attempt rate");
+  declareParameter("Barrier", 1000.0, "coefficient for barrier energy");
```

Comment:
what unit? I assume meV

Reply:
The unit is either K or meV. Depending on which is chosen will affect how the fitting function works.

Reply:
Could you please write that?